config/newsletter.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'apiKey' => env('NEWSLETTER_API_KEY'),
```

Comment:
The default Laravel convention would be `api_key`, but I see we already have a mix.

Reply:
Force of habit, I changed it, because I don't want to add it to the mix.